app/src/main/java/org/fossasia/openevent/general/search/SearchFragment.kt

```diff
@@ -89,6 +91,11 @@ class SearchFragment : Fragment() {
         val thisActivity = activity
         if (thisActivity is MainActivity) searchView = SearchView(thisActivity.supportActionBar?.themedContext)
         searchItem.actionView = searchView
+        searchViewModel.queryText.observe(this, Observer {
+            searchItem.expandActionView()
+            searchView.setQuery(it, false)
+            searchView.maxWidth = Resources.getSystem().displayMetrics.widthPixels
```

Comment:
Why this line?

Reply:
On rotation the search bar is not occupying the whole screen size